src/languages/_french.json

```diff
@@ -541,6 +541,9 @@
         "short_placeholder": "ENS or address :)"
       }
     },
+    "gas": {
+      "network_fee": "Frais de réseau",
+    },
     "homepage": {
```

Comment:
Just wanted to point out that a lot of new keys were added to the english file but not here